staker-rewards/lib/rewards-calculator.js

```diff
@@ -51,12 +51,7 @@ export default class RewardsCalculator {
     )
 
     console.log(
-      clc.yellow(
-        `Rewards weight sum ${rewardWeightSum.toFixed(
-          noDecimalPlaces,
-          BigNumber.ROUND_DOWN
-        )}`
-      )
+      clc.yellow(`Rewards weight sum ${toFormat(rewardWeightSum, false)}`)
```

Comment:
`BigNumber.ROUND_DOWN` should be passed as the third parameter. Otherwise, the default `ROUND_HALF_UP` will be used.